distribution/docker/build.gradle

```diff
@@ -29,7 +32,7 @@ dependencies {
 
 ext.expansions = { Architecture architecture, boolean oss, DockerBase base, boolean local ->
   String classifier
-  if (local) {
+  if (local || base == DockerBase.IRON_BANK) {
```

Comment:
Isn't the iron bank image technically "local". Could we not just pass `local = true` for that build context instead of having this additional condition or does specifying local have other undesired behavior?

Reply:
Ah...that's so obvious now you've pointed it out 🤦